Github/legos/github_get_open_branches/github_get_open_branches.py

```diff
@@ -0,0 +1,64 @@
+
+##
+##  Copyright (c) 2023 unSkript, Inc
+##  All rights reserved.
+##
+from pydantic import BaseModel, Field
+from typing import List
+import pprint
+from github import GithubException
+
+
+class InputSchema(BaseModel):
+    owner: str = Field(
+        description='Username of the GitHub user. Eg: "johnwick"', 
+        title='owner'
+    )
+    repository: str = Field(
+        description='Full name of the GitHub repository. Eg: "unskript/Awesome-CloudOps-Automation"',
+        title='repository',
+    )
+
+
+def github_get_open_branches_printer(output):
+    if not output:
+        return
+    pprint.pprint(output)
+
+
+def github_get_open_branches(handle, owner: str, repository: str)-> List:
+    """github_get_open_branches returns 100 open github branches for a user.
+
+        :type handle: object
+        :param handle: Object returned from task.validate(...).
+    
+        :type owner: string
+        :param owner: Username of the GitHub user. Eg: "johnwick"
+
+        :type repository: string
+        :param repository: Full name of the GitHub repository. Eg: "unskript/Awesome-CloudOps-Automation"
+
+        :rtype: List of branches for a user for a repository
+    """
+    result = []
+    try:
+        user = handle.get_user(login=owner)
+        repos = user.get_repos()
+        repo_name = owner+"/"+repository
```

Comment:
if repository is the full  name (unskript/Awesome-CloudOps-Automation)
does this one make it unskript/unskript/Awesome-CloudOps-Automation?